main.js

```diff
@@ -1,20 +1,83 @@
 import repos from "/data/repos.js";
 
 export default function buildPage() {
-    const root = document.querySelector("div#root");
-    root.innerText = "Hello!";
+    // Creating table
+    let root = document.querySelector("div#root");
+    createTable(root, 4, repos.length);
+    function createTable(parent, cols, rows) {
```

Comment:
it's ok to have `createTable()` functions here, but i would suggest you to make it external. 
and put anywhere else in the file, or you can even put it into a separate js module and import, same as we import repos: 
```js
import repos from "/data/repos.js";
```

Reply:
you don't need `row` as function parameter. 
this function should create table dynamically, without knowing how many records it need to reflect

Reply:
i would suggest you to have this signature for the `createTable()` function:
```js
function createTable(data) {
    // data is the array of repos
    ...
};
```